### PR TITLE
Make direct query to Solr for the file size property which gets stored

### PR DESCRIPTION
**Pull request by narogers**
This should speed up the process of counting byte totals for a collection because it only pulls the file size and only from Solr. You do need to run ActiveFedora::Base.reindex_everything! to add the new file_size_isi field first.

### CODE REVIEW

File: sufia-models/app/models/concerns/sufia/collection_behavior.rb [hunk 1]
```diff
@@ -15,10 +15,29 @@ def update_permissions
...
+      # process
+      args = {
+        :fq => "{!join from=hasCollectionMember_ssim to=id}id:#{self.id}",
+        :fl => "id,file_size_isi",
+        :rows => self.members.count
+      }
+
+      files = ActiveFedora::SolrService.query(query, args)
+
+      byte_count = 0
```
Comment (cam156) [inline sufia-models/app/models/concerns/sufia/collection_behavior.rb]: @narogers These 5 lines could become  files.reduce(0) { |sum, f| sum + f['file_size_isi'] }   Then you would not need the local variable.

**Comment by cam156** [comment]
@narogers it looks like the id of you collection is null when you pass it to solr, so solr is not parsing it.  You can either save the collection in the test (which makes the test slower) or pass and id: to the new operator

**Comment by cam156** [comment]
@narogers it look like there was some issue with the rebase.  Your commit has others in it.

**Comment by narogers** [comment]
I'm not sure what happened so I am just going to close this PR and reopen a new one using the clean branch (without all the commits attached to it)

**Comment by cam156** [comment]
@narogers you are closing this PR correct and creating a new one?

**Comment by narogers** [comment]
Once I find time to rework the spec test I will create a new PR sans all the accidental merging of recent changes to the master branch

**Comment by narogers** [comment]
Closed: See the updated version minus all the extra merges above.

**Pull request closed without merging**